numpy/core/src/multiarray/typeinfo.c

```diff
@@ -104,20 +104,6 @@ PyArray_typeinforanged(
     return entry;
 }
 
-/* Python version only needed for backport to 2.7 */
-#if (PY_VERSION_HEX < 0x03040000) \
-    || (defined(PYPY_VERSION_NUM) && (PYPY_VERSION_NUM < 0x07020000))
```

Comment:
I assumed (from the comment) that PYPY_VERSION_NUM won't be less than 0x07020000?
Can someone confirm

Reply:
I don't think your interpretation of the comment is correct.

However, it might be fine to remove the pypy compatibility here anyway.

Reply:
PyPy 7.2 was released in October, PyPy 7.3 in December. This is still needed for the older version.

Reply:
I updated the comment, thanks for input.